### ISSUE TITLE
Migrate all dgcode-related issues here

### ISSUE THREAD

**Issue by tkittel** [mctools/dgcode#13]
@MilanKlausz I prefer if there is only one issue tracker (i.e. only one place to look) for all dgcode-related issues. Thus, I think we should migrate all issues in the other dgcode_xxx trackers to this one, and afterwards disable the issues in the other projects + leave a clear note about where to go and file issues. Commit message can still refer to the issues if they use the syntax `mctools/dgcode/#1`

**Comment by MilanKlausz** [comment]
Agreed! (Cool! I didn't know about the option to refer to issues in different repos!)
The existing issues are now transferred (from dgcode_actions).
Issues are disabled in: dgcode_actions, dgcode_val, ess-dg/dgcode_projects

- Where can we leave a note about where to create issues? Anything better than the Readme?
-  I'm wondering if it would make sense to allow issues in ess-dg/dgcode_projects, so that users could use it for their projects. Would it be maintainable/usable with labels marking the projects? (I guess it would get entangled in time... On the other hand, it might be useful is we were to make a change in the framework that would require some minor changes in multiple projects...((which I know, shouldn't really happen :)) )

**Comment by tkittel** [comment]
Ah yes, I kind of meant "all dgcode framework and infrastructure related issues" should go here. I think for DG related projects, we should use the ess-dg/dgcode_projects issue tracker.
For where to put the notes... perhaps both the README and the tiny project description which appears at the top right of the project page ?

**Comment by MilanKlausz** [comment]
note: the correct syntax is mctools/dgcode#1
